tough-ssm/src/lib.rs

```diff
@@ -55,12 +55,11 @@ impl KeySource for SsmKeySource {
     fn write(
         &self,
         value: &str,
-        key_id_hex: &str,
     ) -> std::result::Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
         let ssm_client = client::build_client(self.profile.as_deref())?;
         let fut = ssm_client.put_parameter(rusoto_ssm::PutParameterRequest {
             name: self.parameter_name.to_owned(),
-            description: Some(key_id_hex.to_owned()),
+            description: Some(String::from("Tough Key")),
```

Comment:
What's the reason for this change?

Reply:
KeyId added in the root.json file is no longer available with write method because key is created by 'create' method which call the write method. After all this succeeds gen-rsa-key adds the key to root.json.

Reply:
> KeyId added in the root.json file is no longer available with write method because key is created by 'create' method which call the write method. After all this succeeds gen-rsa-key adds the key to root.json.

This seems to be the only change in the tough-ssm crate. How did changes elsewhere affect the behaviour in this crate?